sites/svelte.dev/content/blog/2023-07-01-whats-new-in-svelte-july-2023.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "What's new in Svelte: July 2023"
+description: "Svelte 4.0, New CHANGELOGs and a tour around the community"
```

Comment:
Maybe mention something about all the svelte.dev updates. There's a PR pending with details if you want more info and also some smaller mention in the Svelte 4 post